examples/Zug/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c)
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/cpp-pm/gate
+include("../common.cmake")
+
+project(Zug)
+
+# DOCUMENTATION_START {
+hunter_add_package(Boost)
```

Comment:
Kind of annoying that Boost has to be added manually 😢

Reply:
Yep. I will create an issue at the project...